libs/designsystem/src/lib/components/list/list-virtual-scroll-settings.type.ts

```diff
@@ -0,0 +1,25 @@
+// These types were copied from the vscroll package internals (vscroll/dist/typings/interfaces/settings.d.ts)
+
+export interface VirtualScrollerSettings<T = unknown> {
+  adapter?: boolean;
+  startIndex?: number;
+  minIndex?: number;
+  maxIndex?: number;
+  itemSize?: number;
+  bufferSize?: number;
+  padding?: number;
+  infinite?: boolean;
+  horizontal?: boolean;
+  windowViewport?: boolean;
+  viewportElement?: HTMLElement | (() => void) | null;
+  inverse?: boolean;
+  onBeforeClip?: ItemsProcessor<T> | null;
+}
+
+export declare type ItemsProcessor<T = unknown> = (items: ItemAdapter<T>[]) => void;
```

Comment:
Got some tought on this, easier to talk about tho :)

Reply:
re-exported the settings directly from the lib 👍